docs(refactor): simplify and tidy the run first workflow guide (#90)
canonical/temporal-k8s-operator: documentation/tutorial/08-run-workflow.md

> connected to it, we can trigger a workflow execution and observe its result in

Reviewer: ```suggestion
connected to it, you will trigger a workflow execution and observe its result in
```
Please also update the line above by using second person: "Now that you have a ..."

the Web UI.

> 1. Create simple client to trigger the workflow as defined in the worker.

Reviewer: ```suggestion
1. Create a client to trigger the workflow as defined in the worker.
```

> For instance, if using the Python SDK, this would be:

Reviewer: ```suggestion
For instance, if you're using the Python SDK, this would look like:
```

```python
from temporallib.client import Client, Options
import asyncio

async def main():
    client_opt = Options(
        host="<temporal-host-name>:7233",
        queue="<your-queue>",
        namespace="<your-namespace>",
    )

    client = await Client.connect(client_opt=client_opt)
    workflow_name = "<workflow-name>"
    workflow_id = "<workflow-id>"

    greeting = await client.execute_workflow(
        workflow_name,
        id=workflow_id,
        task_queue="<your-queue>",
    )

if __name__ == "__main__":
    asyncio.run(main())
```

> 2. Install the necessary packages and run the script above using:

Reviewer: ```suggestion
2. Install the necessary packages and run the script above as follows:
```

```bash
pip install temporal-lib-py

python workflow.py
```

3. Further details on the workflow execution can be viewed on the web UI.

[note]

Workflow and activity payloads can be encrypted by setting the same key in
the `auth-secret-id` config of the Temporal worker and the Client configurations
provided by the various Temporal SDKs.
See [`auth-secret-id`](https://charmhub.io/temporal-worker-k8s/configurations#auth-secret-id) for details.

[/note]

[note]

> If TLS termination on ingress is configured, the Certificate Authority (CA) certificate must be also

Reviewer: ```suggestion
If you configure TLS termination on ingress, the Certificate Authority (CA) certificate must be also
```

> If TLS termination on ingress is configured, the Certificate Authority (CA) certificate must be also

Reviewer: ```suggestion
If TLS termination on ingress is configured, you must also provide the Certificate Authority (CA) to the client.
```

> provided to the Client. See [Temporal SDKs](https://docs.temporal.io/develop/) for configuration details.

Reviewer: ```suggestion
See [Temporal SDKs](https://docs.temporal.io/develop/) for configuration details.
```
